pkg/cmd/api-request-count/api_request_count.go

```diff
@@ -0,0 +1,181 @@
+package api_request_count
+
+import (
+	"fmt"
+	"io/ioutil"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"os"
+	"path/filepath"
+	"sort"
+
+	"github.com/ghodss/yaml"
+	"github.com/spf13/cobra"
+
+	apiv1 "github.com/openshift/api/apiserver/v1"
+
+	"k8s.io/klog/v2"
+)
+
+var (
+	example = `
+    # count resources used by users and generate a static HTML page
+    #
+    # "bob": {
+    #  "configmaps":1,
+    #  "secrets":2,
+    # }
+    %[1]s apicount
+
+	# count users of a resource and generate a static HTML page
+    #
+    # "secrets": {
+    #   "bob":1,
+    #   "alice":2,
+    # }
+	%[1]s apicount --by resource
+`
+)
+
+func NewCmdAPIRequestCount(parentName string, streams genericclioptions.IOStreams) *cobra.Command {
+	o := apiRequestCountOptions{}
+
+	cmd := &cobra.Command{
+		Use:          "apicount [flags]",
+		Short:        "Creates a static HTML page for api requests",
+		Example:      fmt.Sprintf(example, parentName),
+		SilenceUsage: true,
+		RunE: func(c *cobra.Command, args []string) error {
+			if err := o.Complete(c, args); err != nil {
+				return err
+			}
+			if err := o.Validate(); err != nil {
+				return err
+			}
+			if err := o.Run(); err != nil {
+				return err
+			}
+
+			return nil
+		},
+	}
+
+	cmd.Flags().StringVar(&o.by, "by", o.by, "Specifies a filter to apply over the original data (eg. -by [user,resource])")
+	cmd.Flags().StringVarP(&o.inputDirectory, "dirname", "d", o.inputDirectory, "A directory which contains api requests data")
```

Comment:
Please use `-f` to be consistent with normal kubectl commands.  they can take directories.